reanalysis/reanalysis_conf.json

```diff
@@ -0,0 +1,91 @@
+{
+    "moi_tests": {
+        "description": "thresholds for different filters during the MOI checks",
+        "gnomad_dominant": 0.001,
+        "gnomad_max_homs_dominant": 1,
+        "gnomad_max_homs_recessive": 10,
+        "gnomad_max_ac_dominant": 100,
+        "class_2_new_only": true
+    },
+    "variant_object": {
+        "description": "variables affecting how the VCF variants are parsed, and AnalysisVariant objects are populated",
+        "csq_string": "Allele|Consequence|SYMBOL|Gene|Feature|MANE_select|BIOTYPE|EXON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|ALLELE_NUM|VARIANT_CLASS|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|GENE_PHENO|SIFT|PolyPhen|LoF|LoF_filter|LoF_flags",
+        "var_info_keep": [
+            "AC",
+            "AN",
+            "cadd",
+            "clinvar_sig",
+            "clinvar_stars",
+            "eigen_phred",
+            "exac_af",
+            "exac_ac_hemi",
+            "exac_ac_het",
+            "exac_ac_hom",
+            "fathmm",
+            "gerp_rs",
+            "gnomad_ac",
+            "gnomad_af",
+            "gnomad_an",
+            "gnomad_cov",
+            "gnomad_hom",
+            "gnomad_ex_ac",
+            "gnomad_ex_af",
+            "gnomad_ex_an",
+            "gnomad_ex_cov",
+            "gnomad_ex_hom",
+            "metasvm",
+            "mutationtaster",
+            "phast_cons",
+            "revel",
+            "slivar_comphet",
+            "splice_ai_csq",
+            "splice_ai_delta"
+        ]
+    },
+    "hail": {
+        "description": "variables for the hail operations, including CSQ sets and filter thresholds",
+        "af_semi_rare": 0.01,
+        "min_samples_to_ac_filter": 70,
+        "ac_filter_percentage": 8,
```

Comment:
Would be great to add descriptions for these parameters, and potentially also a rationale as to why they've been set to a certain value (that's something that's sorely missing e.g. from the gnomAD QC scripts). Not sure if this file is the best place for it though or whether there should be a dedicated section in the docs.

Reply:
added a MarkDown placeholder document, but not yet filler

Reply:
Added a configuration file, but no specific scientific basis for most of the parameters